src/object.rs

```diff
@@ -59,6 +59,11 @@ impl Object {
       .unwrap()
     }
   }
+
+  /// Return the isolate to which the Object belongs to.
+  pub fn get_isolate(&self) -> &mut CxxIsolate {
+    unsafe { v8__Object__GetIsolate(self) }
```

Comment:
This is fine for now, but I think we need a way to return a reference to the public `Isolate` without exposing `CxxIsolate`. I think maybe something like this:
```rust
pub fn get_isolate(&mut self) -> std::cell::RefCell<Isolate>
```
But just leave it as such for now. We can fix this later.